docs/content/en/docs/Customizing/configure_grub.md

```diff
@@ -176,7 +171,7 @@ By default Elemental boots into active, and if there are failures will boot into
 It is possible to override the default fallback logic by setting `default_fallback` as grub environment, consider for example:
 
 ```bash
-> grub2-editenv /oem/grubenv set default_fallback="2 0 1"
+> grub2-editenv /run/cos-state/grub2/grub_oem_env set default_fallback="2 0 1"
```

Comment:
Again, I would limit to document variable meaning and do no instruct to directly change it. The we could have a docs section to modify grubenv vars as an after-install/upgrade hook.